contributing.md

```diff
@@ -67,6 +73,12 @@ and navigate to http://localhost:6060/pkg/github.com/statechannels/go-nitro/
 
 Please execute the following on any branch that's ready for review or merge.
 
+### staticcheck:
+
+```
```

Comment:
not sure if there's a render difference, but consistent with the other shell blocks:

```suggestion
```shell
```